packages/regl-worldview/src/commands/DrawPolygon/index.js

```diff
@@ -72,11 +72,19 @@ type Props = {
   children: DrawPolygonType[],
 };
 
+const polygonLinesGetHitmap: GetHitmap = <T>(props: T, assignNextIds, alreadySeenObjects) => {
```

Comment:
nit: consistent naming throughout please
```suggestion
const polygonLinesGetHitmap: GetHitmap = <T>(props: T, assignNextIds, seenObjects) => {
```